redesign/simple_server.js

```diff
@@ -0,0 +1,14 @@
+var liveServer = require("live-server");
+
+var params = {
+    port: 8181,
+    host: "0.0.0.0",
+    root: "./redesign",
+    open: false,
+    ignore: 'scss,my/templates',
+    file: "./index.html",
+    wait: 1000,
+    mount: [['/components', './node_modules']],
+    logLevel: 2,
+    middleware: [function(req, res, next) { next(); }]
```

Comment:
I've changed this to now look like: 
![image](https://user-images.githubusercontent.com/16775804/26931956-0c6ab8b4-4c5a-11e7-999c-e393ffc5d09c.png)